### ISSUE TITLE
Placeholder and informal names in COL

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#592]
There are 116 names in COL that are considered placeholders, no or informal names:

https://www.checklistbank.org/catalogue/3/names?nameType=no%20name&nameType=informal&nameType=placeholder

I would like to default all sectors to exclude these kind of names in syncs in the future.

Problematic seem to me only the uncertain genus placed names with square brackets. I suppose it would be better to keep these names, but the brackets make them impossible to be processed properly and we should really not encourage that syntax.

**Comment by mdoering** [comment]
@DianRHR @camiplata the sector syncs indeed do not ignore these names by default. The project or sector needs to be configured to ignore these name types. We should do so on dev and prod and override it on a sector if needed - e.g. to include OTU names from trusted sources. The project default should be this: 
<img width="217" alt="image" src="https://github.com/CatalogueOfLife/data/assets/327505/e19285b7-777d-4e8c-9449-bc73fbcaf214">

**Comment by TonyRees** [comment]
Link marked inaccessible to me :(

**Comment by mdoering** [comment]
Yes, the project is not public. But the same names also appear in the latest public release:
https://www.checklistbank.org/catalogue/3LR/names?nameType=no%20name&nameType=informal&nameType=placeholder&offset=0

**Comment by TonyRees** [comment]
Still no access ( code 403) ...

**Comment by mdoering** [comment]
Oh, I had given the wrong URL. Please try this one:
https://www.checklistbank.org/dataset/3LR/names?nameType=no%20name&nameType=informal&nameType=placeholder&offset=0
